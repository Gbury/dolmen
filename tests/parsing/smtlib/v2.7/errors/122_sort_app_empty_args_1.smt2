(set-logic ALL)
(declare-const foo (array _))
