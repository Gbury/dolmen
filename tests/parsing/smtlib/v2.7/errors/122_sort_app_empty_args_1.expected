File "tests/parsing/smtlib/v2.7/errors/122_sort_app_empty_args_1.smt2", line 2, character 26-27:
2 | (declare-const foo (array _))
                              ^
Error while parsing a sort, read the reserved word '_',
      but expected a sort to start a non-empty list of arguments.
